package.json

```diff
@@ -118,23 +120,22 @@
     "d3": "^5.15.0",
     "debounce-stream": "^2.0.0",
     "deep-freeze-strict": "1.1.1",
-    "dnode": "^1.2.2",
+    "dnode": "brave/dnode#remove-weak-package",
```

Comment:
could you address this dependabot alert? https://github.com/brave/dnode/security/dependabot

Reply:
Done.